drivers/hmc5883l/include/hmc5883l_params.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_hmc5883l
+ * @brief       Default configuration for the Honeywell HMC5883L 3-axis digital compass
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ * @{
+ */
+
+#ifndef HMC5883L_PARAMS_H
+#define HMC5883L_PARAMS_H
+
+#include "board.h"
+#include "hmc5883l.h"
+#include "saul_reg.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters
+ * @{
+ */
+#ifndef HMC5883L_PARAM_DEV
+#define HMC5883L_PARAM_DEV          I2C_DEV(0)
+#endif
+#ifndef HMC5883L_PARAM_ADDR
+#define HMC5883L_PARAM_ADDR         (HMC5883L_I2C_ADDRESS)
```

Comment:
The device address is static and can not be changed, so no need of including the addresss in the configuration data set.

Reply:
OK

Reply:
Fixed